libs/stats/docs/pq-cfg.yaml

```diff
@@ -0,0 +1,15 @@
+# Connfiguration file for `odc-stats run` command
+
+# Configure Plugin and Product Attributes
+plugin: pq
+
+# Generic product attributes
+product:
+  name: pc_s2_annual
+  short_name: pc_s2_annual
+  version: 1.0.0
+  product_family: pixel_quality_statistics
+
+  collections_site: explorer.digitalearth.africa
+  producer: digitalearthafrica.org
+  region_code_format: "x{x:03d}y{y:03d}"
```

Comment:
Need more non-default settings

```yaml
s3_public: true
max_processing_time: 600
```

Reply:
I really think this whole file should be inside k8s job definition.

Reply:
Absolutely, I'm about to raise a PR for the job template and embed this in there.